src/main/java/mate/jdbc/service/impl/DriverServiceImpl.java

```diff
@@ -0,0 +1,48 @@
+package mate.jdbc.service.impl;
+
+import java.util.List;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.DriverValidationService;
+
+@Dao
+@Inject
+public class DriverServiceImpl implements DriverService {
+    private DriverDao driverDao;
+    private DriverValidationService validationService;
+
+    public DriverServiceImpl(DriverDao driverDao, DriverValidationService validationService) {
+        this.driverDao = driverDao;
+        this.validationService = validationService;
+    }
+
+    @Override
+    public Driver createDriver(Driver driver) {
+        validationService.validateBeforeCreate(driver);
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Driver getDriverById(Long id) {
+        return driverDao.get(id).orElse(null);
```

Comment:
```suggestion
        return driverDao.get(id).orElse(null);
```
better practice is to throw exception than to return null silently
```suggestion
        return driverDao.get(id).orElseThrow(() -> new NoSuchElementException("Couldn't find a driver with id: " + id));
```